docs/source/whitepaper.rst

```diff
@@ -335,20 +334,82 @@ Charles can attempt to delay Bob's withdrawal by challenging Bob's rightful clai
 is sufficient to cover Bob's fee for L1 resolution, Bob will proceed with L1 resolution. If not, Bob can continue opening
 parallel claims until Charles no longer contests one of them, or there is enough accumulated stake from Charles on the
 multiple challenges for Bob to do an L1 resolution. In any case, Bob will be able to prove his rightful claim before
-Charles' claim reach the end of its period.
-The time constraint described in the previous example also holds in this case.
+Charles' claim reach the end of its period. The time constraint described in the previous example also holds in this case.
 
 Claims that cannot be filled
 ++++++++++++++++++++++++++++
 
 In both the regular `false claim` and `self-challenge claim` cases, we assumed that Bob could fill Alice's request in
 order to prove that the false claimer Charles was not the correct filler. However, If Alice's request cannot be filled
 for any reason (e.g. transfer value too high), instead of proving that someone other than Charles filled a request,
```

Comment:
```suggestion
order to prove that the false claimer Charles was not the correct filler. However, Alice's request might not be able to be filled for any reason (e.g. transfer value too high). Instead of proving that someone other than Charles filled a request,
```